tests/neg/i11994.check

```diff
@@ -1,26 +1,12 @@
--- [E008] Not Found Error: tests/neg/i11994.scala:2:28 -----------------------------------------------------------------
-2 |implicit def foo[T <: Tuple.meow]: Unit = ??? // error
-  |                      ^^^^^^^^^^
-  |                      type meow is not a member of object Tuple.
-  |                      Extension methods were tried, but the search failed with:
-  |
-  |                          Cyclic reference involving method foo
-  |                          
-  |                          The error occurred while trying to compute the signature of method foo
-  |                            which required to compute the signature of type T
-  |                            which required to compute the signature of method foo
-  |                          
-  |                           Run with both -explain-cyclic and -Ydebug-cyclic to see full stack trace.
--- [E008] Not Found Error: tests/neg/i11994.scala:3:18 -----------------------------------------------------------------
+-- [E046] Cyclic Error: tests/neg/i11994.scala:3:18 --------------------------------------------------------------------
 3 |given [T <: Tuple.meow]: Unit = ??? // error
-  |            ^^^^^^^^^^
-  |            type meow is not a member of object Tuple.
-  |            Extension methods were tried, but the search failed with:
+  |                  ^
+  |                  Cyclic reference involving method given_Unit
```

Comment:
`Tuple.meow` is always illegal. This message is more useless than before and not showing this fact.

Reply:
Indeed. It was a problem in the error reporting that caused the real error to be hidden.